### DISCUSSION TITLE
Did not receive wSRM

### DISCUSSION THREAD

**Discussion by allmails888** [wormhole-foundation/wormhole#236] [Problems / bugs] [answered]
Had no issue previously transferring SRM on ethereum to wSRM on solana until now. my transaction hash is https://etherscan.io/tx/0x8f6e721f6d32cba05a500e23300b3ea084312d978e4a89d9232df8402b702d76. eth transaction was confirmed with >15 confirmations.  appreciate any assistance
![image](https://user-images.githubusercontent.com/17685229/126604469-2ad9fa08-4db4-4811-8b00-192c741b6286.png)
p/s: after a while, the final msg on wormhold ui says "transfer completed on solana" but the wSRM is still missing from my specified receiving wallet at GtewVicRqsrQTN2jwnEBqy97T6fEcHAN13Xsq6LaH5X5

**Comment by hendrikhofstadt** [comment, marked as answer]
Completed in https://explorer.solana.com/tx/4crceRpwjT42XatpyghRWGivCEKasjRuyswbdu3byEFNXUAPasQC46NJK7TmDmzRYwdXmhn3DpKuxgfpZQ9tLhsn

**Reply by allmails888** [reply]
saw the transaction finally at solana side. thank you

am hesitant to use wormhole again due to risk of this happening again. any indication on reason for the delays and if we should expect to see more of such instances going forward?

**Reply by hendrikhofstadt** [reply]
We actually don’t recommend using Wormhole v1 (except for bridging assets back). We strongly advised protocols to hold back on pointing users to it because active development of it has stopped months ago and we are currently working on v2 which will come with significant improvements (e.g. faster transfers and no more delays). V2 should be arriving in the next weeks.

**Reply by allmails888** [reply]
Thanks so much for responding to this. well noted on this point. cant wait for v2 to be released. cheers mate
